traitsui/wx/range_editor.py

```diff
@@ -518,19 +512,17 @@ def update_object_on_scroll(self, event):
                 if self.factory.is_float:
                     self.value = value
                 else:
-                    self.value = int(value)
+                    self.value = int( value )
         finally:
             self.ui_changing = False
 
-    #-------------------------------------------------------------------------
+    #---------------------------------------------------------------------------
     #  Handle the user pressing the 'Enter' key in the edit control:
-    #-------------------------------------------------------------------------
+    #---------------------------------------------------------------------------
 
-    def update_object_on_enter(self, event):
+    def update_object_on_enter ( self, event ):
         """ Handles the user pressing the Enter key in the text field.
         """
-        if isinstance(event, wx.FocusEvent):
-            event.Skip()
```

Comment:
Behavioural change: This block is removed.

I think this block is needed though.